src/editors/containers/VideoEditor/components/VideoSettingsModal/components/TranscriptWidget/LanguageSelector.jsx

```diff
@@ -54,27 +59,57 @@ export const LanguageSelector = ({
   const onLanguageChange = module.hooks.onSelectLanguage({
     dispatch: useDispatch(), languageBeforeChange: localLang, setLocalLang, triggerupload: input.click,
   });
+  console.log({ localLang, language, openLanguages });
+
+  const getTitle = () => {
+    if (Object.prototype.hasOwnProperty.call(videoTranscriptLanguages, language)) {
+      return (
+        <ActionRow>
+          {videoTranscriptLanguages[language]}
+          <ActionRow.Spacer />
+          <Icon className="text-primary-500" src={Check} />
```

Comment:
Is there a reason for the check here? There isn't one in the mocks, only when you are looking at the dropdown list.

Reply:
mmm I can give it a fix later if people get mad. Some paragon selects have them when you have an item selected.